packages/core/src/App/app.jsx

```diff
@@ -63,30 +64,45 @@ const AppWithoutTranslation = ({ root_store }) => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const checkInputClick = React.useCallback(() => {
+        if (document.activeElement.tagName === 'INPUT' && isMobile() && isSafari()) {
+            root_store.ui.setIsInputClicked(true);
+        } else {
+            root_store.ui.setIsInputClicked(false);
+        }
+    }, [root_store.ui]);
```

Comment:
The landscape-blocker would appear if the keyboard appears when the user clicks on an input box, and they are on safari. This is happening in prod/staging. 

Needed to check if the user has clicked input, if they are on mobile and if they are using safari. Once the keyboard appears, it'll resize the viewport, but since I added this check in the handleResize if statement, it will not show the landscape-blocker.